editor/src/clj/editor/gl/texture.clj

```diff
@@ -419,21 +578,15 @@
 
 (scene-cache/register-object-cache! ::texture make-texture update-texture destroy-textures)
 
-(def ^:private cubemap-targets
-  {:px GL/GL_TEXTURE_CUBE_MAP_POSITIVE_X
-   :nx GL/GL_TEXTURE_CUBE_MAP_NEGATIVE_X
-   :py GL/GL_TEXTURE_CUBE_MAP_POSITIVE_Y
-   :ny GL/GL_TEXTURE_CUBE_MAP_NEGATIVE_Y
-   :pz GL/GL_TEXTURE_CUBE_MAP_POSITIVE_Z
-   :nz GL/GL_TEXTURE_CUBE_MAP_NEGATIVE_Z})
```

Comment:
Moved up in the file so we can use it to validate the arguments to `make-gpu-cubemap-texture`.